src/modules/Dashboard/components/Dashboard/style.scss

```diff
@@ -0,0 +1,10 @@
+.text-gray {
```

Comment:
You need to make that these styles are applied only in the `Dashboard` component. Otherwise, they will be applied in other components. The same remark for other classes.